images/Makefile

```diff
@@ -39,6 +48,11 @@ build-rootfs:
 		else \
 			echo "Building rootfs image: $$image"; \
 			docker build -t $$image $$rootfs_dir; \
+			if [ "$(PUSH)" = "true" ]; then \
+				echo "Pushing image: $$image"; \
+				docker push $$image; \
+			fi; \
+			docker image rm -f $(docker images -a | grep -v "initrd-builder:0.0.1" | awk 'NR>1 {print $3}'); \
```

Comment:
Clear up all images except for `initrd-builder:0.0.1` that is not pushed anywhere and is only used locally as a base for other images.